tests/test_hook.py

```diff
@@ -32,6 +33,7 @@
 TOKEN_OWNER_URL = f"{USERS_URL}/{TOKEN_OWNER_ID}"
 RUN_AFTER_HOOK_ID = 999
 RUN_AFTER_URLS = [f"{HOOKS_URL}/{RUN_AFTER_HOOK_ID}"]
+EXTRA_SPACE_IN_CLICK7 = " " if int(click.__version__.split(".")[0]) < 8 else ""
```

Comment:
`click` will now be always packed with a version higher than 8, so this shouldn't be needed, I suppose?

Reply:
Right now I made the change compatible with both click 7 and 8 - and in setup.py it's configured as `click<8.1.0`. We can change it to `click==8.0.*` and get rid of compatibility hacks, but that would force all users to use click 8. So this way it's more permissive e.g. for someone who has other dependencies requiring older click, but it's a bit more complex for us.

I'll let the decision up to you, I don't have a strong opinion on this and I'm not familiar w/ how others use this package.